src/metaScene.cxx

```diff
@@ -168,16 +168,16 @@ MetaScene::Read(const char * _headerName)
     const std::string objectType = MET_ReadType(*m_ReadStream);
     if (!strncmp(objectType.c_str(), "Tube", 4) || ((objectType.empty()) && !strcmp(suf, "tre")))
     {
-      char * subtype = MET_ReadSubType(*m_ReadStream);
-      if (!strncmp(subtype, "Vessel", 6))
+    const std::string subtype = MET_ReadSubType(*m_ReadStream);
+      if (!strncmp(subtype.c_str(), "Vessel", 6))
```

Comment:
Why not switch this to a regular string comparison? e.g. `if (subtype == "Vessel")`. Perhaps add an additional style commit?

Reply:
Doh!  Was following old pattern in the file.  I'll update this and other tests.  Thanks!